tutorials/subdomain_and_interface_projections.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>Suggested rewrite, to make this appear a bit less like black magic:</p><p>Specifically, we have the following grid-relations (not visible from the plots, but available through inspection) which we will use later:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>